app/lifecycle/server.go

```diff
@@ -18,11 +18,17 @@ func getCLIFullPath(command string) string {
 	var cmdPath string
 	appExe, err := os.Executable()
 	if err == nil {
+		// Check both the same location as the tray app, as well as ./bin
 		cmdPath = filepath.Join(filepath.Dir(appExe), command)
 		_, err := os.Stat(cmdPath)
 		if err == nil {
 			return cmdPath
 		}
+		cmdPath = filepath.Join(filepath.Dir(appExe), "bin", command)
```

Comment:
Would the `ollama` cli be in either directory?

Reply:
Just a little safeguarding for potentially converging to a common layout between windows and the other OSes where we put the CLI in ./bin relative to the "install dir". It searches both locations so it's safe